_discsimmodule.c

```diff
@@ -338,8 +350,9 @@ Simulator_init(Simulator *self, PyObject *args, PyObject *kwds)
     int sim_ret;
     static char *kwlist[] = {"sample", "event_classes", "num_loci", 
             "num_parents", "max_population_size", "max_occupancy", 
-            "dimension", "simulate_pedigree", "random_seed", "torus_diameter",
-            "pixel_size", "recombination_probability", NULL}; 
+            "dimension", "simulate_pedigree", "simulate_kingman", 
+            "random_seed", "torus_diameter", "pixel_size", 
+            "recombination_probability", "r", "Ne", NULL}; 
```

Comment:
We are breaking the pattern a bit here with the ``r`` and ``Ne`` parameters. The meaning of everything else is spelled out pretty literally. Something like ``arg_recombination_rate`` and ``arg_effective_population_size`` would be more appropriate for the Python interface I think.

Do we need two different recombination rates, or is r always divided by Ne? If so, can we perhaps remove the r parameter entirely from the interface?